docs/guide/topics/audit_logs.rst

```diff
@@ -0,0 +1,134 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_audit_logs:
+
+Audit Logs
+============
+
+Audit logs log various administrative actions taken in a guild. Examples include:
+
+- When a channel is created, updated, or deleted
+- When channel-specific permissions are created, updated, or deleted
+- When a member is kicked, pruned, banned, unbanned, updated, has their roles changed, or is moved or disconnected from a voice channel
+- When a bot is added to the guild
+- When a role is created, updated, or deleted
+- When an invite is created, updated, or deleted
+- When a webhook is created, updated, or deleted
+- When an emoji is created, updated, or deleted
+- When a message is deleted, bulk-deleted, pinned, or unpinned
+- When a thread is created, updated, or deleted
+    
+Depending on the action taken, an entry in the audit log may contain additional info about that specific action, including:
+
+
+
+.. warning:: 
+
+    There are a couple of limitations with audit logs that need to be understood:
+
+    1. There is no guarantee on audit logs arriving when expected (if they arrive at all)
+    2. No event is triggered when an audit log entry is created.
```

Comment:
There is now an event that's called for said thing; [`on_audit_log_entry_create`](https://discordpy.readthedocs.io/en/latest/api.html#discord.on_audit_log_entry_create)